class ResponsePracticumException(Exception):
    """Исключение из-за ошибок в запросе к сервису."""

    pass
